### PR TITLE
Closes #74 bug running pkgdown build site admiralneuro.rmd

### LINKED ISSUE #74: Bug: running pkgdown::build_site() due to admiralneuro.Rmd
### What happened?

While running `pkgdown::build_site()`, the file `ad_adapet.R` is created in the `vignettes` subfolder. Something might be wrong, please double-check the reason.

Image

When re-running `pkgdown::build_site()`the following error is created:

Image

**The reason:**
Please ensure that the chunks from `admiralneuro.Rmd` at lines 38 and 51 are using `, eval = FALSE` as in below example:

Image

### Session Information

R version 4.2.3 (2023-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: CentOS Linux 7 (Core)

Matrix products: default
BLAS/LAPACK: /usr/lib64/libopenblasp-r0.3.3.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8   
 [6] LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] admiralneuro_0.1.0         metatools_0.1.6            stringr_1.5.1              lubridate_1.9.4            dplyr_1.1.4               
[6] pharmaversesdtm_1.3.0.9002 admiral_1.3.1.9003         admiraldev_1.3.1.9002      testthat_3.2.3            

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.14       tidyr_1.3.1       ps_1.9.1          rprojroot_2.1.0   digest_0.6.37     mime_0.13         R6_2.6.1          cellranger_1.1.0 
 [9] evaluate_1.0.4    pillar_1.11.0     rlang_1.1.6       readxl_1.4.5      rstudioapi_0.17.1 miniUI_0.1.2      callr_3.7.6       urlchecker_1.0.1 
[17] rmarkdown_2.29    pkgdown_2.1.3     desc_1.4.3        devtools_2.4.5    htmlwidgets_1.6.4 shiny_1.11.1      compiler_4.2.3    httpuv_1.6.15    
[25] xfun_0.52         pkgconfig_2.0.3   pkgbuild_1.4.8    htmltools_0.5.8.1 tidyselect_1.2.1  tibble_3.3.0      roxygen2_7.3.2    withr_3.0.2      
[33] later_1.4.2       brio_1.1.5        commonmark_2.0.0  xtable_1.8-4      lifecycle_1.0.4   magrittr_2.0.3    cli_3.6.5         metacore_0.2.0   
[41] stringi_1.8.7     cachem_1.1.0      fs_1.6.6          promises_1.3.2    remotes_2.5.0     xml2_1.3.8        ellipsis_0.3.2    generics_0.1.4   
[49] vctrs_0.6.5       tools_4.2.3       glue_1.8.0        purrr_1.1.0       hms_1.1.3         processx_3.8.6    pkgload_1.4.0     fastmap_1.2.0    
[57] yaml_2.3.10       timechange_0.3.0  sessioninfo_1.2.3 memoise_2.0.1     knitr_1.50        profvis_0.4.0     usethis_3.1.0 

### Reproducible Example

devtools::load_all()
pkgdown::build_site()

### PR DESCRIPTION
**Pull request by Lina2689**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: inst/WORDLIST [hunk 1]
```diff
@@ -1,64 +1,44 @@
 ADAPET
```
Comment (manciniedoardo) [inline inst/WORDLIST:1]: Are all these updates needed/intended?

File: vignettes/admiralneuro.Rmd [hunk 2]
```diff
@@ -48,7 +48,7 @@ use_ad_template(
 A list of all available templates can be obtained by `list_all_templates()`
 from `{admiral}`:
   
-```{r}
+```{r, eval = FALSE}
 list_all_templates(package = "admiralneuro")
```
Comment (manciniedoardo) [inline vignettes/admiralneuro.Rmd:52]:
```suggestion
```{r, eval = TRUE}
admiral::list_all_templates(package = "admiralneuro")
```

This way the site still renders but you can see the available templates
image

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks! Left some suggestions

File: inst/WORDLIST (hunk 1 above)
Comment (manciniedoardo) [reply inst/WORDLIST:1]: I think if you don't edit the file, you won't have any conflicts.

**Comment by manciniedoardo** [comment]
Thanks @Lina2689!
